docs/proposals/YYYYMMDD-template.md

```diff
@@ -48,22 +17,15 @@ superseded-by:
 ---
 
 # Title
+- Keep it simple and descriptive.
+- A good title can help communicate what the proposal is and should be considered as part of any review.
 
-This is the title of the proposal.
-Keep it simple and descriptive.
-A good title can help communicate what the proposal is and should be considered as part of any review.
-
-The title should be lowercased and spaces/punctuation should be replaced with `-`.
-
+<!-- Remove before PR -->
 To get started with this template:
 1. **Make a copy of this template.**
   Copy this template into `docs/enhacements` and name it `YYYYMMDD-my-title.md`, where `YYYYMMDD` is the date the proposal was first drafted.
-1. **Fill out the Summary and Motivation sections.**
+1. **Fill out the required sections.**
 1. **Create a PR.**
-1. **Merge early.**
```

Comment:
I don't necessarily like the idea of removing the `Merge early` concept, otherwise we are likely to get hung up on bikeshedding before getting proposals in.

Reply:
In the timeline below, I expressed that proposals ideas should be communicated in an issue or community meeting for initial feedback. Originally, the merge early was to get just motivation and goals in place, which so far hasn't been enough to get consensus.